nicegui/testing/user_download.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+import asyncio
+from pathlib import Path
+from typing import TYPE_CHECKING, Any, List, Optional, Union
+
+import httpx
+
+from .. import background_tasks
+
+if TYPE_CHECKING:
+    from .user import User
+
+
+class UserDownload:
+
+    def __init__(self, user: User) -> None:
+        self.http_responses: List[httpx.Response] = []
+        self.user = user
+
+    def __call__(self, src: Union[str, Path, bytes], filename: Optional[str] = None, media_type: str = '') -> Any:
+        background_tasks.create(self._get(src))
+
+    async def _get(self,  src: Union[str, Path, bytes]) -> None:
+        if isinstance(src, bytes):
+            await asyncio.sleep(0)
+            response = httpx.Response(200, content=src)
```

Comment:
Do you want to consider using `HTTP_200_OK`?

Reply:
Sure!